ogr/ogrsf_frmts/openfilegdb/filegdbtable.cpp

```diff
@@ -2557,13 +2558,56 @@ int FileGDBTable::GetIndexCount()
             ReadUTF16String(pabyCur, nIdxNameCarCount));
         pabyCur += 2 * nIdxNameCarCount;
 
+        // 4 "magic fields"
+        returnErrorAndCleanupIf(static_cast<GUInt32>(pabyEnd - pabyCur) <
```

Comment:
I do not follow: if the comparison is between a sum of `size_t` (that I assume to be `size_t` itself) why cast to GUInt32 ?

Reply:
addressed in https://github.com/OSGeo/gdal/pull/11410